store/use-room-store.ts

```
@@ -0,0 +1,60 @@
+  removeFromCart: (index) => set((state) => ({
+    cart: state.cart.filter((_, i) => i !== index)
+  })),
+  
+  clearCart: () => set({ cart: [] }),
+  
+  // Computed
+  getCartTotal: () => {
+    const { cart } = get();
+    return cart.reduce((total, cartItem) => {
+      const priceStr = cartItem.item.price || '0€';
+      const price = parseFloat(priceStr.replace(/[€,]/g, '').replace(',', '.'));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)